core/src/forwarding_stage.rs

```diff
@@ -115,14 +115,22 @@ impl ForwardAddressGetter {
     }
 }
 
+/// [`SpawnForwardingStageResult`] contains the result of spawning the
```

Comment:
nit: maybe better to document individual fields rather than larger comment for whole struct? Would be easier to keep docs in sync with code.

Reply:
I think it is more ideomatic to describe the purpose of the whole structure rather than fields. For example, by looking into https://doc.rust-lang.org/stable/src/alloc/vec/mod.rs.html#397